### PR TITLE
IEP-478: Application size analysis view hardcoded strings

### PR DESCRIPTION
**Pull request by alirana01**
Application size analysis view hardcoded strings converted to the properties files

### CODE REVIEW

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java [hunk 1]
```diff
@@ -114,9 +116,12 @@ private void plotSingleBar()
 		long available_diram = (long) overviewJson.get(IDFSizeConstants.AVAILABLE_DIRAM);
 		double used_diram_ratio = (double) overviewJson.get(IDFSizeConstants.USED_DIRAM_RATIO);
 
-		toolkit.createLabel(overviewComp, "Used DIRAM:"); //$NON-NLS-1$
-		String chartText = convertToKB(used_diram) + Messages.IDFSizeOverviewComposite_0 + convertToKB(available_diram) + " available, "  //$NON-NLS-1$
-				+ Math.round(used_diram_ratio * 100) + "%"+ "used"+")";  //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+		toolkit.createLabel(overviewComp, Messages.IDFSizeOverviewComposite_SinglePlot_UsedDiram);
+//		String chartText = convertToKB(used_diram) + Messages.IDFSizeOverviewComposite_0 + convertToKB(available_diram)
+//				+ " available, " //$NON-NLS-1$
+//				+ Math.round(used_diram_ratio * 100) + "%" + "used" + ")"; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```
Comment (kolipakakondal) [inline bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java:122]: Could you remove this commented code?

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java [hunk 2]
```diff
@@ -135,16 +140,22 @@ protected void plotDoubleBar()
 		double used_dram_ratio = (double) overviewJson.get(IDFSizeConstants.USED_DRAM_RATIO);
 
 		// Used static DRAM
-		toolkit.createLabel(overviewComp, "Used static DRAM:"); //$NON-NLS-1$
-		String dramText = convertToKB(used_dram) + " (" + convertToKB(available_dram) + " available, "  //$NON-NLS-1$ //$NON-NLS-2$
-				+ Math.round(used_dram_ratio * 100) + "%"+ " used"+")";  //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+		toolkit.createLabel(overviewComp, Messages.IDFSizeOverviewComposite_UsedStaticDram);
+//		String dramText = convertToKB(used_dram) + " (" + convertToKB(available_dram) + " available, "  //$NON-NLS-1$ //$NON-NLS-2$
+//				+ Math.round(used_dram_ratio * 100) + "%"+ " used"+")";  //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```
Comment (kolipakakondal) [inline bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java:145]: Could you remove this commented code?

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java [hunk 3]
```diff
@@ -135,16 +140,22 @@ protected void plotDoubleBar()
...
 		Label dramUsedVal = toolkit.createLabel(overviewComp, dramText); // $NON-NLS-1$
 		dramUsedVal.setFont(boldFont);
 
 		// Used static IRAM
-		toolkit.createLabel(overviewComp, "Used static IRAM:");  //$NON-NLS-1$
-		String iramText = convertToKB(used_iram) + " (" + convertToKB(available_iram) + " available, " //$NON-NLS-1$ //$NON-NLS-2$ 
-				+ Math.round(used_iram_ratio * 100) + "%" + " used"+ ")"; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+		toolkit.createLabel(overviewComp, Messages.IDFSizeOverviewComposite_UsedStaticIram);
+//		String iramText = convertToKB(used_iram) + " (" + convertToKB(available_iram) + " available, " //$NON-NLS-1$ //$NON-NLS-2$ 
+//				+ Math.round(used_iram_ratio * 100) + "%" + " used"+ ")"; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```
Comment (kolipakakondal) [inline bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java:156]: Could you remove this commented code?

**Review by kolipakakondal — APPROVED** [review]
LGTM

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java (hunk 1 above)
Comment (alirana01) [reply bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java]: Done

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java (hunk 2 above)
Comment (alirana01) [reply bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java]: Done

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java (hunk 3 above)
Comment (alirana01) [reply bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeOverviewComposite.java]: Done

**Pull request merged**